timestamp/src/token2/graphql.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use super::state_machine::GetAccount;
+use super::types::*;
+use super::{ServiceHandler, StateMachine};
+use crate::common::*;
+pub use ckey::{Ed25519Private as Private, Ed25519Public as Public};
+use coordinator::module2::*;
+use remote_trait_object::Service;
+use std::sync::Arc;
+
+struct GraphQlRoot {
+    state_machine: StateMachine,
+}
+
+#[async_graphql::Object]
+impl GraphQlRoot {
+    async fn account(&self, public: GqlPublic) -> Option<Account> {
+        self.state_machine
+            .execute_access(GetAccount {
+                public: &public.0,
+                default: false,
+            })
+            .ok()
+    }
+}
+
+#[async_graphql::Object]
+impl Token {
+    async fn issuer(&self) -> String {
+        hex::encode(self.issuer.as_ref())
```

Comment:
Can't we use GqlH256 here?

Reply:
We are not using it but might be need if we add another query.

Reply:
I don't understand the context. I think you added `GqlH256` to use it here.
Is there any other usecase of `GqlH256`?

Reply:
No I'm not using it in this PR, but it might be needed if we add a new query that **takes** H256. (I still don't know how to return it) This is a separated commit, so I can simply drop it if you want.

Reply:
Aha, I understand it.